### ISSUE TITLE
Create a YAML File for Machine Learning Models

### ISSUE THREAD

**Issue by slavakurilyak** [produvia/kryptos#76]
# Goal
As a developer, I want to create a YAML file to define machine learning model configurations, so that I can easily modify parameter and hyperparameter settings.

As a developer, I want to create a YAML file to define machine learning model configurations, so that I can find the optimal parameter and hyperparameter settings.

# Consider
- Consider creating a config flag via: `$ strat -ml xgboost -config hyperparam.yaml`

```yaml
# hyperparam.yaml
trainingInput:
  scaleTier: STANDARD_1
  hyperparameters:
    maxTrials: 40
    maxParallelTrials: 5
    enableTrialEarlyStopping: True
    goal: MINIMIZE    
    hyperparameterMetricTag: rmse
    params:
    - parameterName: nembeds
      type: INTEGER
      minValue: 3
      maxValue: 30
      scaleType: UNIT_LINEAR_SCALE
    - parameterName: nnsize
      type: DISCRETE
      discreteValues:
      - 4
      - 16
      - 32
      - 64
      - 128
```

# Inspiration

Google is using YAML files to configure their machine learning models. ([Google, 2018](https://cloud.google.com/blog/big-data/2018/03/hyperparameter-tuning-on-google-cloud-platform-is-now-faster-and-smarter))

> You will identify parts of your model that you want the tuner to change so as to optimize the evaluation metric. For example, you might want the tuner to be able to tune the batch size and the number of embedding nodes in your model. All of these hyperparameters will have to be command-line inputs to your executable Python package. You’d then put these together into a YAML file.
